src/Components/Facility/TriageForm.tsx

```diff
@@ -242,19 +243,13 @@ export const TriageForm = (props: triageFormProps) => {
     });
   };
 
-  const handleChange = (e: any) => {
-    const form = { ...state.form };
-    form[e.target.name] = e.target.value;
-    dispatch({ type: "set_form", form });
-  };
-
   if (isLoading) {
     return <Loading />;
   }
 
   return (
     <div className="px-2">
-      <PageTitle
+      <Page
```

Comment:
Why is the `Page` component wrapping only the Modal? Shouldn't it wrap the entire page instead?

Page component gives consistent designs like content padding and positions for title, page options, etc.